### DISCUSSION TITLE
Is projection performed by grid_from_radars?

### DISCUSSION THREAD

**Discussion by mattyd100** [ARM-DOE/pyart#1514] [General]
Background: I am using pyart to generate a cartesian grid of reflectivity data. I then use this grid to estimate precipitation intensity and 'adjust' it based on ground-based rain gauge data. 

Question: When I create a grid using grid_from_radars and I am using only one radar, is the grid projected onto the curved surface of the earth? Or do the x,y coordinates of the grid represent coordinates in a flat plane, oriented perpendicular to the z-axis and centered at the radar location? 

The reason I am asking is that I need to I need to make sure that I know which grid cells correspond with the rain gauge locations. My rain gauges are contained in a GIS layer and their coordinates are projected.

I looked through the code and I see that you can call grid_from_radars with a grid_projection parameter which gets passed to map_to_grid. But it seems that if you only have one radar and grid_origin_alt and grid_origin  are not specified, then skip_transform is set to True and the projection is not used to calculate the cartesian coordinates for the gates. I apologize if I am missing something obvious.

**Comment by mgrover1** [comment]
The coordinates are projected onto an Azimuthal equal area projection. You will then have your x/y coordinates, as well as your latitude and longitude, but the latitude/longitude are technically two dimensional (not flat).
